web/packages/teleport/src/services/version/unsupported.ts

```diff
@@ -32,6 +32,26 @@ import { JoinToken, JoinTokenRequest } from '../joinToken';
 
 export const ProxyRequiresUpgrade = 'Ensure all proxies are upgraded';
 
+/**
+ * Determines if error 404 is a result of an endpoint path not found
+ * (not resource not found).
+ */
+export function isPathNotFoundError(err: unknown) {
```

Comment:
What's this change about? It doesn't seem related to the CSRF removals I made.

Reply:
this is used to add fallback support if the new UI goes to a older proxy.

i can also push this change into another PR?

Reply:
Yeah another PR would be good I think as keeping this PR scoped to "partial-revert" will make it easy to reference later.